src/odemis/acq/align/autofocus.py

```diff
@@ -57,6 +57,27 @@ def _convertRBGToGrayscale(image):
     return gray
 
 
+def AssessFocus(levels):
+    """
+    Given a list of focus levels, it decides if there is any significant value
+    or it only contains noise.
+    returns (boolean): True if there is significant deviation
+    """
+    max_l = max(levels)
+    min_l = min(levels)
+    levels.remove(max_l)
+    levels.remove(min_l)
```

Comment:
Explain why you remove min_l, and never care about it anymore (or don't remove it)